plugin-libs/plugin-model/src/main/java/com/salesforce/bazel/eclipse/model/BazelPackageLocation.java

```diff
@@ -0,0 +1,43 @@
+package com.salesforce.bazel.eclipse.model;
+
+import java.io.File;
+
+/**
+ * Minimal representation of a Bazel Package location on the file system.
+ * 
+ * @see BazelLabel for a container that is also Bazel Target aware. 
+ * @author stoens
+ * @since March 2020
+ */
+public interface BazelPackageLocation {
```

Comment:
I copied the important methods, needed during import, from BazelPackageInfo into this interface.  I considered changing the method names a little, but left them alone for now to make it more obvious what is going on.  Rename these can be a different PR.